.github/workflows/release.yml

```diff
@@ -25,24 +25,22 @@ jobs:
         with:
           go-version-file: 'go.mod'
           cache: true
-      # TODO: allow signing artifacts (see Card #75)
       # This uses an action (hashicorp/ghaction-import-gpg) that assumes you set your
       # private key in the `GPG_PRIVATE_KEY` secret and passphrase in the `PASSPHRASE`
       # secret. If you would rather own your own GPG handling, please fork this action
       # or use an alternative one for key handling.
-      #      - name: Import GPG key
-      #        id: import_gpg
-      #        uses: hashicorp/ghaction-import-gpg@v2.1.0
-      #        env:
-      #          # These secrets will need to be configured for the repository:
-      #          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-      #          PASSPHRASE: ${{ secrets.PASSPHRASE }}
+      - name: Import GPG key
+        id: import_gpg
+        uses: hashicorp/ghaction-import-gpg@v2.1.0
+        env:
+          # These secrets will need to be configured for the repository:
```

Comment:
Do we need to change this comment to reflect the status of this repository?

Reply:
Yep, good catch.